pkg/agent/proxy/proxier.go

```diff
@@ -53,6 +54,9 @@ type Proxier interface {
 	// flows and the OVS group IDs for a Service. False is returned if the
 	// Service is not found.
 	GetServiceFlowKeys(serviceName, namespace string) ([]string, []binding.GroupIDType, bool)
+	// GetServiceByIP returns the ServicePortName struct of giving serviceString(ClusterIP:Port/Proto).
+	// False is returned if the serviceString is not existed in serviceStringMap.
```

Comment:
s/is not existed/does not exist
or
s/is not existed/is not found

Reply:
Thanks, addressed.